add music skill for playing songs

gkuznik opened the pull request:
Tested with Spotify, fixes #83 
Only potential issue is confusion with the media skill that also uses play. For now both work, with *play media* taking precedence.

gkuznik committed:
add music skill for playing songs

Stypox reviewed (changes requested):
Looks mostly good to me, thank you!

Also add the skill information to the README and to the English fastlane file.

gkuznik committed:
address review

add intent docs
add README + fastlane skill description
change music skill strings

gkuznik committed:
improve intent building
